### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by yigitm**
In this PR:

- #index & #show views are created for user & post models
- model methods that are created in past milestone are used for view operations
- partials are used in views
- basic styling is added

Thank you for your time.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,25 @@
...
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+
+  <%= render 'shared/recent_posts' %>
+ 
+  <button>
+  <%= link_to 'See all posts', user_posts_path(@user) %>
+  </button>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb:13]: - [ ] This link is supposed to take us to all posts of the user it means it has to display all posts but currently I am still getting only 3 recent posts please address that

File: app/views/shared/_single_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+     <p class='post-title'><%= @post.title %> by <%= @post.author.name %></p>
+   
+    <div class='post-counters'>
+     <p class='post-c-counter'>Comments: <%= @post.comments_counter %></p>
+     <p class='post-l-counter'>Likes: <%= @post.likes_counter %></p>
+   </div>
+  </div>
+ 
+   <p class='post-text'><%= @post.text %></p>
+  </div>
```
Comment (DeliceLydia) [inline app/views/shared/_single_post.html.erb:12]: - [ ] When I check a post page it is displaying only 5recent comments but instead, it is supposed to display all comments, it should display only 5recent comments when you are on the page of all user posts

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @yigitm, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Kudos to the good README
- Followed git workflow
- No linter errors

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/views/users/show.html.erb (hunk 1 above)
Comment (yigitm) [reply app/views/users/show.html.erb:13]: Thank you for that. It fixed. I missed to implementing it.

File: app/views/shared/_single_post.html.erb (hunk 2 above)
Comment (yigitm) [reply app/views/shared/_single_post.html.erb:12]: Thank you for that. I fixed the single user posts in order to display all posts instead recent ones.

**Review by bimbolabuari — APPROVED** [review]
Hi @yigitm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**